File: CSCI 161/Midterm/Program 3.py
'''
Adam Roy
CSCI 161
MIDTERM Program 3
'''

def clamp(val):
    if val < 0:
        return 0
    elif val >= 0 and val <= 255:
        return val
    elif val > 255:
        return 255

def main():
    inp = int(input("Enter a number: "))
    f = clamp(inp)
    print(f)
main()
        

File: CSCI 280/Assignment 03/Program3.java
/*
CSCI280
Adam Roy
Program: 3
Due: Feb 1st 2023
*/

public class Program3{
    public static void main(String[] args){
        Student info1 = new Student();
        info1.enrollNumber = 5;
        info1.studentName = ("    John             ");
        Student info2 = new Student();
        info2.enrollNumber = 6;
        info2.studentName = ("    Bruce            ");
        System.out.println("Student Name | Enroll Number");
        System.out.println(info1.studentName + info1.enrollNumber);
        System.out.println(info2.studentName + info2.enrollNumber);
    }}

class Student{
    String studentName;
    int enrollNumber;}